programs/drift/src/instructions/lp_pool.rs

```diff
@@ -1454,6 +1462,26 @@ pub fn handle_deposit_to_program_vault<'c: 'info, 'info>(
         "Constituent balance mismatch after withdraw from program vault"
     )?;
 
+    let new_token_balance = constituent
+        .spot_balance
+        .get_signed_token_amount(&spot_market)?
+        .cast::<i64>()?;
+
+    emit!(LPBorrowLendDepositRecord {
+        ts: clock.unix_timestamp,
+        slot: clock.slot,
+        spot_market_index: spot_market.market_index,
+        constituent_index: constituent.constituent_index,
+        direction: DepositDirection::Deposit,
+        token_balance: new_token_balance,
+        last_token_balance: constituent.last_spot_balance_token_amount,
+        interest_accrued_token_amount: bl_position_after_interest_update
+            .safe_sub(bl_position_before_interest_update)?
+            .cast::<i64>()?,
+        amount_deposit_withdraw: amount,
+    });
+    constituent.last_spot_balance_token_amount = new_token_balance;
+
```

Comment:
good to add a `constituent.cumulative_interest_accrued_token_amount` to track lifetime earn/paid